src/library/searchquery.cpp

```diff
@@ -107,6 +107,10 @@ NotNode::NotNode(QueryNode* pNode)
         : m_pNode(pNode) {
 }
 
+NotNode::~NotNode() {
+    delete m_pNode;
```

Comment:
Both the GroupNode and the NotNode own the pointers of their component(s). But only the GroupNode deletes them in the destructor.